SimCalorimetry/PhaseIIAnalyzer/plugins/PhaseIIAnalyzer.cc

```diff
@@ -0,0 +1,243 @@
+// -*- C++ -*-
+//
+// Package:    PhaseII/PhaseIIAnalyzer
+// Class:      PhaseIIAnalyzer
+//
+/**\class PhaseIIAnalyzer PhaseIIAnalyzer.cc PhaseII/PhaseIIAnalyzer/plugins/PhaseIIAnalyzer.cc
+   Description: [one line class summary]
+   Implementation:
+   [Notes on implementation]
+*/
+//
+// Original Author:  Dario Soldi
+//         Created:  Tue, 16 Jan 2018 11:56:05 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+//My includes
+#include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
+#include "CondFormats/EcalObjects/interface/EcalPedestals.h"
+#include "CondFormats/DataRecord/interface/EcalPedestalsRcd.h"
+#include "CondFormats/EcalObjects/interface/EcalLiteDTUPedestals.h"
+#include "CondFormats/DataRecord/interface/EcalLiteDTUPedestalsRcd.h"
+#include "CondFormats/DataRecord/interface/EcalIntercalibConstantsMCRcd.h"
+#include "CondFormats/EcalObjects/interface/EcalIntercalibConstantsMC.h"
+
+#include "PhaseIIAnalyzer.h"
+
+#include "DataFormats/EcalDetId/interface/EBDetId.h"
+#include "DataFormats/EcalDetId/interface/EEDetId.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "SimDataFormats/CaloHit/interface/PCaloHit.h"
+
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include "SimDataFormats/Track/interface/SimTrack.h"
+#include "CondFormats/EcalObjects/interface/EcalConstants.h"
+
+using namespace std;
+using namespace edm;
+
+//
+// constants, enums and typedefs
+//
+
+//
+// static data member definitions
+//
+
+//
+// constructors and destructor
+//
+
+PhaseIIAnalyzer::PhaseIIAnalyzer(const edm::ParameterSet& iConfig) {
```

Comment:
see comments on PhaseIAnalyzer:
* move to test
* combine h and cc
* delete commented-out code

Reply:
actually, I had not initially realized these had their own packages. that seems unnecessary. just put them all in SimCalorimetry/EcalSimAlgos/test